_targets.R

```diff
@@ -21,24 +24,27 @@ list(
              ),
 
   # TODO: PULL SITE DATA HERE
-  tar_target(
-    wi_data,
-    get_site_data(sites_info = oldest_active_sites,
-                  state = "WI",
-                  parameter = parameter)
-  ),
-  tar_target(
-    mn_data,
-    get_site_data(sites_info = oldest_active_sites,
-                  state = "MN",
-                  parameter = parameter)
-  ),
-  tar_target(
-    mi_data,
-    get_site_data(sites_info = oldest_active_sites,
-                  state = "MI",
-                  parameter = parameter)
+  tar_map(
+    values = tibble(state_abb = states),
+    tar_target(
+      nwis_data,
+      get_site_data(sites_info = oldest_active_sites,
+                    state = state_abb,
+                    parameter = parameter))
   ),
+    # ),
+    # tar_target(
+    #   count,
+    #   get_site_data(sites_info = oldest_active_sites,
+    #                 state = state_abb,
+    #                 parameter = parameter)
+    # ),
+    # tar_target(
+    #   fig,
+    #   get_site_data(sites_info = oldest_active_sites,
+    #                 state = state_abb,
+    #                 parameter = parameter)
+    # ),
```

Comment:
This can be deleted since it's not in use

Reply:
Got it!